lib/publiccloud/utils.pm

```diff
@@ -49,4 +49,24 @@ sub is_ondemand() {
     return is_publiccloud && (!is_byos());    # When introducing new flavors, add checks here accordingly.
 }
 
+# Check if we are an Azure test run
+sub is_azure() {
+    return is_publiccloud && get_var('PUBLIC_CLOUD_PROVIDER') == "AZURE";
+}
+
+# Check if we are an Google Compute Engine test run
+sub is_gce() {
+    return is_publiccloud && get_var('PUBLIC_CLOUD_PROVIDER') == "GCE";
+}
+
+# Check if we are an AWS test run
+sub is_ec2() {
+    return is_publiccloud && get_var('PUBLIC_CLOUD_PROVIDER') == "EC2";
+}
+
+# Check if we are an AWS test run
+sub is_aws() {
+    return is_ec2();
+}
```

Comment:
I won't do that otherwise you may introduce `is_microsoft()` and similar 😆

Reply:
Don't get me started :laughing: